<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>435</width>
    <height>298</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>127.0.0.1</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumberMin">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>90</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSliderMin">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>70</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Min</string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSliderMax">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumberMax">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>140</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>120</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Max</string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSliderTiming">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>180</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>35</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Timing(s)</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>201</width>
      <height>221</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>170</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string> 1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonStart">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonStop">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>210</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QLabel" name="status">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonConnect">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonDisconnect">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>40</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Disconnect</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>435</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>horizontalSliderMax</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumberMax</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>146</x>
     <y>164</y>
    </hint>
    <hint type="destinationlabel">
     <x>161</x>
     <y>177</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderMin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumberMin</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>101</x>
     <y>114</y>
    </hint>
    <hint type="destinationlabel">
     <x>190</x>
     <y>126</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderTiming</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_4</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>136</x>
     <y>213</y>
    </hint>
    <hint type="destinationlabel">
     <x>194</x>
     <y>212</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
